lib/lambdaws.js

```diff
@@ -31,6 +31,8 @@ var relativeDir = path.dirname(module.parent.parent.filename);
 
 function _resolvePathFromParent(p) {
     try {
+        // Deps with prepending ":" are external dependencies. We do not resolve them; we process them differently
+        if(p[0] === ':') return p;
```

Comment:
to be even more picky, you could extract this prepending external dep prefix in a global var...

that way, you could remove all the comments related to ' : ' and later would be much easier to work